agent/agent-tooling/src/main/java/com/microsoft/applicationinsights/agent/internal/init/SecondEntryPoint.java

```diff
@@ -248,6 +271,140 @@ public void customize(AutoConfigurationCustomizer autoConfiguration) {
         .addShutdownHook(new Thread(() -> flushAll(telemetryClient).join(10, TimeUnit.SECONDS)));
   }
 
+  private static TelemetryItemExporter initStatsbeatTelemetryItemExporter(
```

Comment:
Should this method be migrated into the export to allow stat beat to use the file system?